p/scripts/main.js

```diff
@@ -581,6 +581,15 @@ function collapse_entry() {
 	toggleContent(flux_current, flux_current, false);
 }
 
+function toggle_media() {
+	const media = document.querySelector('.flux.current video,.flux.current audio');
+	if (media.paused) {
+		media.play();
+	} else {
+		media.pause();
+	}
```

Comment:
```suggestion
	const media = document.querySelector('.flux.current video,.flux.current audio');
	if (media) {
		if (media.paused) {
			media.play();
		} else {
			media.pause();
		}
	}
```

Reply:
We need to check that `media` is not null

Reply:
Fixed